patch_root_roadmap.sh

```
@@ -0,0 +1 @@
+sed -i 's/The active planning state is now selection-first:/The active planning state is now selection-first:\n\n- PR evidence packet workflows (v1.14) are complete./g' ROADMAP.md
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)